Utilities/GenerateSecurityGraphs/GenerateSecurityGraphs/Program.cs

```diff
@@ -546,6 +543,18 @@ private static List<Resource> GetRootNodes(AdjacencyGraph<Resource, Edge<Resourc
             return subgraph;
         }
 
+        private static Resource AddAndGetVertex(AdjacencyGraph<Resource, Edge<Resource>> resourceGraph, string claimName)
+        {
+            var vertex = resourceGraph.Vertices.SingleOrDefault(v => v.Name == claimName);
+            if (vertex is null)
```

Comment:
Add blank lines between blocks (between `var vertex` and `if (vertext is null)`.

Reply:
Thank you; done!